virus/Lineari_obscurus.md

```diff
@@ -0,0 +1,106 @@
+# **Obscurity turned outward: hidden residents of the mouse gut microbiome**
+
+written by: [Meijin Wu]
+
+### {Q1B Virus Name}
```

Comment:
Amend out the "Default Titles" from the document. Your descriptive titles are much better throughout.